frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init

```diff
@@ -51,13 +49,12 @@ set_config_expr() {
 
 prepare_run_dir() {
   mkdir -p "${RUN_DIR}"
-  chown "${orchestrator_user}":cvdnetwork "${RUN_DIR}"
-  chmod 775 "${RUN_DIR}"
+  chown "${USER}" "${RUN_DIR}"
 }
 
 start_orchestrator() {
   mkdir -p  "${orchestrator_cvd_artifacts_dir}"
-  chown "${orchestrator_user}":cvdnetwork "${orchestrator_cvd_artifacts_dir}"
+  chown "${USER}:${USER}" "${orchestrator_cvd_artifacts_dir}"
```

Comment:
`chown "${USER}": "${orchestrator_cvd_artifacts_dir}"` is a more flexible option as the group `${USER}` may not exists. See `man chown`.

Reply:
Done.